syntax = "proto3";

message PlayerReadyC2S {
    
}